### PR TITLE
Translate guides/directory-structure/nuxt-config.md page to spanish

### PR DESCRIPTION
**Pull request by Kronhyx**
Translate the following pages to Spanish (es-ES) #543 :

:memo: guides -> directory structure
- [x] nuxt-config.md

### CODE REVIEW

**Comment by Kronhyx** [comment]
@debs-obrien .... PR ready 4 translation review

File: content/es/guides/directory-structure/nuxt-config.md [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+
+No es necesario colocar la extensión para los archivos CSS/SCSS/Postcss/Less/Stylus/... enumerados en el arreglo css en el archivo `nuxt.config`.
+
+```js{}[nuxt.config.js]
+export default {
+  css: ['~/assets/css/main', '~/assets/css/animations']
+}
+```
+
+Al omitir la extensión, si tiene un archivo css y decides cambiar para usar sass, por ejemplo, no es necesario el `nuxt.config` ya que usará la nueva extensión una vez que el nombre del archivo siga siendo el mismo.
```
Comment (Kronhyx) [inline content/es/guides/directory-structure/nuxt-config.md:78]:
```suggestion
Al omitir la extensión, si tiene un archivo css y decides cambiar para usar sass, por ejemplo, no es necesario actualizar el `nuxt.config` ya que usará la nueva extensión una vez que el nombre del archivo siga siendo el mismo.
```

File: content/es/guides/directory-structure/nuxt-config.md [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+
+<base-alert type="next">
+
+Aprende más sobre [dev property](/guides/configuration-glossary/configuration-dev)
+
+</base-alert>
+
+### env
+
+Esta opción te permite definir variables de entorno que están disponibles tanto para el cliente como para el servidor.
```
Comment (Kronhyx) [inline content/es/guides/directory-structure/nuxt-config.md:105]:
```suggestion
Esta opción te permite definir variables de entorno que estarán disponibles tanto para el cliente como para el servidor.
```

**Comment by debs-obrien** [comment]
Great work. @alvarosaburido or @Dawntraoz if either of you get a chance to give it a quick review that would be awesome

**Comment by debs-obrien** [comment]
Great work @Kronhyx 😊

**Pull request merged**